ES/ESamq003.xml

```diff
@@ -397,8 +397,8 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                   <provenance>The name(s) originally mentioned in the
                      supplication formulas have been erased (e.g., <locus target="#11va">11va</locus> l.15, 11vb l.16, 16ra l.24) and mostly replaced with the name
                      Tasfā Māryām, written in a secondary hand (e.g., <locus target="#4rb">4rb</locus> l.2, 4va ls.12-13). Tasfā Māryām is the baptismal name of
-                     daǧǧāzmāč Kasa (s. Additio 1), a son of
-                     daǧǧāzmāč Subagadis
+                     daǧǧāzmāč Kāsā (s. Additio 1), a son of
+                     daǧǧāzmāč Subagādis
```

Comment:
elsewhere Subāgādis

Reply:
here and elsewhere, i did not really take much care to standardize all names, it should be a separate workflow, standardize and link, i wanted to make the records visible first

Reply:
I understand...I just personally find inconsistent transliteration hard to ignore :)